### PR TITLE
Update the doc about how to setup on Alibaba Cloud

### PR DESCRIPTION
**Pull request by osswangxining**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
@brian-avery The two failing tests are known issues for which we have a release blocking issue open.

**Comment by osswangxining** [comment]
> @brian-avery The two failing tests are known issues for which we have a release blocking issue open.

so can we skip this known issues for this PR or any other suggestions?  @brian-avery

**Comment by osswangxining** [comment]
> @brian-avery The two failing tests are known issues for which we have a release blocking issue open.

@ericvn @brian-avery  any suggestions to follow up this PR? Thanks.

File: content/en/docs/setup/platform-setup/alicloud/index.md [hunk 1]
```diff
@@ -11,8 +11,6 @@ owner: istio/wg-environments-maintainers
 test: n/a
 ---
 
-This page was last updated August 8, 2018.
-
 {{< boilerplate untested-document >}}
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/alicloud/index.md:14]: The boilerplate has this text: `This vendor-provided document has not been tested on the Istio 1.9 release and may contain bugs.`  If you feel that is accurate, you can leave it in, else you could remove it.

Comment (osswangxining) [reply content/en/docs/setup/platform-setup/alicloud/index.md:14]: Yes, we can leave it in and go ahead. Thanks for your reply.

**Comment by osswangxining** [comment]
@ericvn  what I need to add for this PR to be merged? thanks for your guide!

**Review by ericvn — APPROVED** [review]

**Pull request merged**